api/src/main/java/net/jqwik/api/Arbitrary.java

```diff
@@ -441,8 +441,10 @@ public boolean isUnique() {
 	@API(status = MAINTAINED, since = "1.3.0")
 	default T sample() {
 		return this.sampleStream()
-				   .findFirst()
-				   .orElseThrow(() -> new JqwikException("Cannot generate a value"));
+			.map(Optional::ofNullable)
```

Comment:
I would be a little bit concerned about performance... On the other side, this method should not be called extensively. It is called only by user while experimenting, or perhaps from "normal" JUnit tests. Am I right?